### ISSUE TITLE
Update of APIs and Regulations

### ISSUE THREAD

**Issue by NTFLeo** [aaassseee/stripe_flutter_channel#2]
The Stripe APIs are going to change soon, This will require the use PaymentIntents it is possible with this library or will be possible before the 14 of September?

<img width="624" alt="Schermata 2019-07-29 alle 15 40 39" src="https://user-images.githubusercontent.com/25492442/62054145-4b5fb200-b219-11e9-9354-855281a5fc8b.png">

Useful link:
PaymentIntent API

**Comment by aaassseee** [comment]
Yeah, sure. But I am still working on the integration on Stripe with Google
Pay and Apple Pay which may take some time. However I will try my best to
complete the changes before deadline.

**Comment by ezet** [comment]
I had the same issues and ended up using a slightly different approach. Feel free to test it out https://pub.dev/packages/stripe_sdk
